### DISCUSSION TITLE
Where is the root of your Wiki.js installation? Docker/Ubuntu

### DISCUSSION THREAD

**Discussion by tracer8** [requarks/wiki#3843] [Help / Questions] [answered]
Hi, 
I'm using Docker/Ubuntu install.  I want to edit config.yaml of wiki.js.
So where is the root of your Wiki.js installation?

**Comment by Phil-hostwiki** [comment, marked as answer]
To access the container, run `docker exec -it wiki bash`.

After editing, restart the docker container for changes to take effect.

**Comment by tracer8** [comment]
Thank you, i got it now.
